megadoc/.obsidian/app.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Nitpick: did you intend to push all of the `.obsidian` files?

Reply:
No--just meant to push the literacy_images file and the unit-01.md file.

Reply:
Great! I didn't look too far into the unit change, but everything seems okay to me!